samza-core/src/main/java/org/apache/samza/execution/JobPlanner.java

```diff
@@ -149,25 +125,6 @@ final void writePlanJsonFile(String planJson) {
     }
   }
 
-  // TODO: SAMZA-1814: the following configuration generation still misses serde configuration generation,
-  // side input configuration, broadcast input and task inputs configuration generation for low-level task
-  // applications
-  // helper method to generate a single node job configuration for low level task applications
-  private JobConfig prepareTaskJob(TaskApplicationDescriptorImpl taskAppDesc) {
-    // copy original configure
-    Map<String, String> cfg = new HashMap<>();
-    // expand system and streams configure
-    Map<String, String> systemStreamConfigs = expandSystemStreamConfigs(taskAppDesc);
-    cfg.putAll(systemStreamConfigs);
-    // expand table configure
-    cfg.putAll(expandTableConfigs(cfg, taskAppDesc));
-    // adding app.class in the configuration
-    cfg.put(ApplicationConfig.APP_CLASS, appDesc.getAppClass().getName());
```

Comment:
Just to double check: Is this config no longer necessary?

Reply:
After consolidation, this configuration is always added in getExecutionPlan() method.